baseball.py

```diff
@@ -0,0 +1,47 @@
+
+from pyspark import StorageLevel
+from pyspark.ml.feature import CountVectorizer
+from pyspark.sql import SparkSession
+
+# Setup Spark
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+database = "baseball"
+port = "3306"
+user = "root"
+password = "root"
```

Comment:
Looks like you didn't run the pre-commit hooks. If it's giving errors, add # pragma: allowlist secret to the same line where you have password.

Reply:
!!!!